pkg/schema/load/load_test.go

```diff
@@ -123,3 +123,20 @@ func TestPanelValidity(t *testing.T) {
 		return nil
 	}))
 }
+
+func TestCueErrorWrapper(t *testing.T) {
+	t.Run("Testing scuemata validity with valid cue schemas", func(t *testing.T) {
+		tempDir := os.DirFS(filepath.Join("testdata", "malformed_cue"))
+
+		var baseLoadPaths = BaseLoadPaths{
+			BaseCueFS:       tempDir,
+			DistPluginCueFS: GetDefaultLoadPaths().DistPluginCueFS,
+		}
+
+		_, err := BaseDashboardFamily(baseLoadPaths)
+		require.EqualError(t, err, "errors: [\"missing ',' in struct literal\" \"expected '}', found 'EOF'\"], in file: /cue/data/gen.cue, on line: 13")
```

Comment:
Having this specified down to the line number seems a bit brittle. Not necessary to change, but i'm curious if you can loosen the criteria to make maintenance easier, without being concerned about increasing false positive or negative rate

Reply:
Yeah, probably something that matches the beginning of the error message should do it.

Reply:
Not sure i'm really following. But having a smaller, simpler file (per my other new comment) will maybe make this less important.

Reply:
Yeah didn't make that clear. I thought you meant that it's difficult to be maintained the way it is now - code wise. I now _think_ I understood that by _make maintenance_ easier you mean only having the simplest file - test wise? Updated!

Reply:
Ah, i think i see the confusion. Sorta moot with a smaller file, but i'll explain.

This is theory-of-validation stuff, again, which reduces to:

1. What do we know if it passes? Can it ever not pass when it really should? (false positive)
2. What do we know if it fails? Can it ever fail if it really shouldn't? (false negative)

My maintenance point was mostly about the false negatives: if that test fails because something trivial is changed in the file - e.g., a newline is added or removed before line 13, then the test will fail as it's originally written. At that point, it's all about "should it have failed?"

I would argue that it should not have - something changed (whitespace) in the test inputs that was materially irrelevant to the property the test is intended to demonstrate (invalid CUE syntax causes errors), but the test is still failing. Sure, in this case, it's easy to figure out and fix, but test reliability - and therefore, good test design amounts to writing tests such that variations irrelevant to the property under test cannot cause false positives or negatives.